### PR TITLE
Documentation for configuration files

### LINKED ISSUE #13: Document how the config works

### PR DESCRIPTION
**Pull request by Tansito**
Fix #13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 # pwa-lit-template
 
-##### [Getting started](#getting-started) | [Build for production](#build-for-production) | [Create a new page](#create-a-new-page) | [Browser support](#browser-support)
+##### [Getting started](#getting-started) | [Build for production](#build-for-production) | [Create a new page](#create-a-new-page) | [Configuration files](#configuration-files) | [Browser support](#browser-support)
```
Comment (abdonrd) [inline README.md:10]: Personally y prefer just `Configuration` instead of `Configuration files`.

**Review by abdonrd — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Tansito) [reply README.md:10]: Could be but maybe too generic? Maybe something more like `Environment configuration`?

Comment (abdonrd) [reply README.md:10]: But it really is the only configuration we will have, right?

`Environment configuration` also works.

Comment (Tansito) [reply README.md:10]: Well not at all, without a context we could talk about the configuration of typescript or rollup to put an example (in fact, `configuration files` is even worse now that this example came to my mind 😂 ).

Ok, let's change it to `Environment configuration` it fits better.

**Review by abdonrd — APPROVED** [review]

**Pull request merged**